createdisk-library.sh

```diff
@@ -158,7 +158,7 @@ function prepare_cockpit() {
 function prepare_hyperV() {
     local vm_ip=$1
 
-    install_additional_packages ${vm_ip} hyperv-daemons
+    install_additional_packages ${vm_ip} hyperv-daemons cifs-utils
```

Comment:
👍🏼

Reply:
I am wondering if this needs to be generic for 'all' images, or just on Windows.
CIFS shares are not only usable on Windows.

For the current PR ofcourse it is, but this does not have to be the case.

WDYT?

Reply:
don't have any preference in this case, on one hand since there is a more performant `virtiofs` option available users won't need CIFS, but if they can't use virtiofs for some reason and want to use CIFS we are preventing them, but to support CIFS for all the platforms we'll need more work on `crc`

i think when we do the necessary work on `crc` to make `CIFS` available to all platforms we can include this package for all then?

Reply:
> i think when we do the necessary work on crc to make CIFS available to all platforms we can include this package for all then?

makes sense!